core/src/main/java/net/sourceforge/jnlp/util/LocaleUtils.java

```diff
@@ -47,40 +49,38 @@ public static Locale getLocale(final String localeString) throws ParseException
      * @see Locale
      * @see Match
      */
-    public static boolean localeMatches(Locale requested, Locale[] available, Match matchLevel) {
+    public static boolean localeMatches(final Locale requested, Locale[] available, final Match matchLevel) {
+        Assert.requireNonNull(requested, "requested");
```

Comment:
matchLevel must not be `null` otherwise the switch statement will throw a NPE

Reply:
done